web/ui/templates/targets.html

```diff
@@ -1,57 +1,72 @@
-{{define "head"}}<!-- nix -->{{end}}
+{{define "head"}}
+<link type="text/css" rel="stylesheet" href="{{ pathPrefix }}/static/css/targets.css?v={{ buildVersion }}">
```

Comment:
What's the reason to break this out into another css file? The overhead of another HTTP roundtrip to fetch the file should be higher than to the few bytes added to the file even when not looking at the targets page.